src/main/java/bio/terra/cli/businessobject/Workspace.java

```diff
@@ -332,6 +357,10 @@ public String getGoogleProjectId() {
     return googleProjectId;
   }
 
+  public Properties getProperty() {
```

Comment:
getProperties. Please try to pay attention to singular (property) vs plural (properties). :)